lms/djangoapps/teacher_dashboard/urls.py

```diff
@@ -1,21 +1,26 @@
 """
 URLs for the Labster Teacher Dashboard.
 """
-from django.conf.urls import patterns, url
+from django.conf.urls import patterns, url, include
 
-urlpatterns = patterns(
-        '',
-        url(r'^teacher_dashboard/?$', 'teacher_dashboard.views.dashboard_view', name='dashboard_view_handler'),
 
-        # Api calls
-        url(r'^licenses/?$', 'teacher_dashboard.views.licenses_api_call'),
-        url(r'^licenses/(?P<license_pk>[a-zA-Z0-9\-]+)/simulations/?$', 'teacher_dashboard.views.simulations_api_call'),
-        url(
-            r'^licenses/(?P<license_pk>[a-zA-Z0-9\-]+)/simulations/(?P<simulation_pk>[a-zA-Z0-9]+)/students/?$',
-            'teacher_dashboard.views.students_api_call'
-        ),
-        url(
-            r'^licenses/(?P<license_pk>[a-zA-Z0-9\-]+)/simulations/(?P<simulation_pk>[a-zA-Z0-9]+)/attempts/?$',
-            'teacher_dashboard.views.attempts_api_call'
-        ),
+teacher_dashboard_patterns = patterns(
+    '',
+    url(r'^licenses/?$', 'teacher_dashboard.views.licenses_api_call'),
+    url(r'^licenses/(?P<license_pk>[a-zA-Z0-9]+)/simulations/?$', 'teacher_dashboard.views.simulations_api_call'),
+    url(
+        r'^licenses/(?P<license_pk>[a-zA-Z0-9]+)/simulations/(?P<simulation_pk>[a-zA-Z0-9]+)/students/?$',
+        'teacher_dashboard.views.students_api_call'
+    ),
+    url(
+        r'^licenses/(?P<license_pk>[a-zA-Z0-9]+)/simulations/(?P<simulation_pk>[a-zA-Z0-9]+)/attempts/?$',
+        'teacher_dashboard.views.attempts_api_call'
+    ),
+)
+
+urlpatterns = patterns(
+    '',
+    url(r'^teacher_dashboard/?$', 'teacher_dashboard.views.dashboard_view', name='dashboard_view_handler'),
+    # Api calls
+    url(r'^teacher/api/v0/', include(teacher_dashboard_patterns)),
```

Comment:
@polesye this api is really per course, because it needs course id as parameter, correct?